.github/workflows/ci.yml

```diff
@@ -65,3 +65,10 @@ jobs:
         env:
           PYTHON_VERSION: ${{ matrix.python-version }}
           NOX_SESSION: ${{ matrix.nox-session }}
+          # TEMPORARY for 3.11
+          # https://github.com/aio-libs/aiohttp/issues/6600
+          AIOHTTP_NO_EXTENSIONS: 1
+          # https://github.com/aio-libs/frozenlist/issues/285
+          FROZENLIST_NO_EXTENSIONS: 1
+          # https://github.com/aio-libs/yarl/issues/680
+          YARL_NO_EXTENSIONS: 1
```

Comment:
This solution was taken from https://github.com/python/bedevere/pull/399/commits/eccd492b2e07e1e3fbb2e1367038f97cba0c8748